catalog/app/components/Preview/load.js

```diff
@@ -30,9 +31,10 @@ const loaderChain = [
   Pdf,
   Vcf,
   Html,
-  Text,
```

Comment:
`Text` handles audio format `.ts`. And, I think Text should be at the end of list, because it's the last resort to render file :)

I think, we need a switcher for `.ts`, the same as for Voila/Notebook

Reply:
not sure about switch, probably we can do just fine without typescript previews)